mikeio/generic.py

```diff
@@ -69,35 +70,38 @@ def _clone(infilename, outfilename):
 
 
 def scale(
-    infilename, outfilename, offset=0.0, factor=1.0, item_numbers=None, item_names=None
+    infilename: str, outfilename: str, offset: float = 0.0,
+    factor: float = 1.0, items: Union[None, List[str], List[int]] = None,
 ):
     """Apply scaling to any dfs file
 
         Parameters
         ----------
 
-        infilename: str
+        infilename:
```

Comment:
why remove the type from the docstring? Isn't this useful for the user also when you use type hints?

Reply:
Maybe, I figured it was redundant, but there is no reason to remove them as long as it is consistent with the type hint.

Reply:
I think it's still relevant for the user - but maybe I missed something